content/docs/intro/cloud-providers/docker/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Docker
+meta_desc: This page provides an overview of the Docker Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-docker
+    weight: 5
+
+aliases: ["/docs/reference/clouds/docker/"]
```

Comment:
This alias is not needed and should be removed (this page didn't exist previously at that URL path).

Reply:
done!